js/src/modal.js

```diff
@@ -35,13 +36,15 @@ const ESCAPE_KEY = 'Escape'
 const Default = {
   backdrop: true,
   keyboard: true,
-  focus: true
+  focus: true,
+  rootElement: 'body'
 }
 
 const DefaultType = {
   backdrop: '(boolean|string)',
   keyboard: 'boolean',
-  focus: 'boolean'
+  focus: 'boolean',
+  rootElement: '(element|string)'
```

Comment:
I think is enough as you resolve it before type check
```suggestion
  rootElement: 'element'
```

Reply:
This is also the case in `backdrop.js` should I remove `string` from there also?

Reply:
I would suggest not to mess with backdrop at all for now 😉  (act like it is fine, in order to avoid open two many things at once)

Reply:
https://github.com/twbs/bootstrap/pull/34149 is merged now, should I update this?